.jenkins/Jenkinsfile

```diff
@@ -22,517 +22,447 @@
 // Validate/lint this file using the following command
 // `curl -X POST  -F "jenkinsfile=<.jenkins/Jenkinsfile" https://ci-cassandra.apache.org/pipeline-model-converter/validate`
 
+def jdksSupported = ["11", "17"] 
+def archsSupported = ["amd64", "arm64"]
+def pythonsSupported = ["3.7", "3.8", "3.11"] 
+def jdkDefault = "11"
+def pythonDefault = "3.8"
+def stageResults = [:]
+def splits = 10
+
+// Steps config
+def stepsMap = [
+  'jar': [type: 'build', script: 'build-jars.sh', toCopy: null],
+  'artifacts': [type: 'build', script: 'build-artifacts.sh', toCopy: 'apache-cassandra-*.tar.gz,apache-cassandra-*.jar,apache-cassandra-*.pom'],
+  'lint': [type: 'build', script: 'check-code.sh', toCopy: null],
+  'debian': [type: 'build', script: 'build-debian.sh', toCopy: 'cassandra_*,cassandra-tools_*'],
+  'redhat': [type: 'build', script: 'build-redhat.sh rpm', toCopy: '*.rpm'],
+  'cqlsh-test': [type: 'test', splits: false],
+  'fqltool-test': [type: 'test', splits: false],
+  'test-cdc': [type: 'test', splits: splits],
+  'test': [type: 'test', splits: splits],
+  'test-trie': [type: 'test', splits: splits],
+  'test-compression': [type: 'test', splits: splits],
+  'stress-test': [type: 'test', splits: splits],
+  'test-burn': [type: 'test', splits: splits],
+  'long-test': [type: 'test', splits: splits],
+  'test-oa': [type: 'test', splits: splits],
+  'test-system-keyspace-directory': [type: 'test', splits: splits],
+  'jvm-dtest': [type: 'test', splits: splits, dtest: true],
+  'jvm-dtest-upgrade': [type: 'test', splits: splits, dtest: true],
+  'dtest': [type: 'test', splits: splits, dtest: true],
+  'dtest-novnode': [type: 'test', splits: splits, dtest: true],
+  'dtest-offheap': [type: 'test', splits: splits, dtest: true],
+  'dtest-large': [type: 'test', splits: splits, dtest: true],
+  'dtest-large-novnode': [type: 'test', splits: splits, dtest: true],
+  'dtest-upgrade': [type: 'test', splits: splits, dtest: true],
+  'dtest-upgrade-novnode': [type: 'test', splits: splits, dtest: true],
+  'dtest-upgrade-large': [type: 'test', splits: splits, dtest: true],
+  'dtest-upgrade-novnode-large': [type: 'test', splits: splits, dtest: true],
+]
+
+// define matrix axes
+def Map matrix_axes = [
+    arch: archsSupported,
```

Comment:
just "arm" for now

Reply:
You meant `amd`, right? I'll remove it from the list `archsSupported` as it should be "the single source of truth"

Reply:
Done